shared/server-commands/videos/channel-syncs-command.ts

```diff
@@ -0,0 +1,66 @@
+import { HttpStatusCode, ResultList, VideoChannelCreateResult, VideoChannelSync, VideoChannelSyncCreate } from "@shared/models"
+import { pick } from "lodash"
+import { unwrapBody } from "../requests"
+import { AbstractCommand, OverrideCommandOptions } from "../shared"
+
+export class ChannelSyncsCommand extends AbstractCommand {
+  private static readonly API_PATH = '/api/v1/video-channel-syncs'
+  listByAccount (options: OverrideCommandOptions & {
+    accountName: string
+    start?: number
+    count?: number
+    sort?: string
+  }) {
+    const { accountName, sort = 'createdAt' } = options
+    const path = `/api/v1/accounts/${accountName}/video-channel-syncs`
+    return this.getRequestBody<ResultList<VideoChannelSync>>({
+      ...options,
+
+      path,
+      query: { sort, ...pick(options, [ 'start', 'count', 'withStats', 'search' ]) },
```

Comment:
You don't need `withStats` and `search`

Use `pick` from `@shared/core-utils` that is strictly typed instead of the one of lodash